README.md

```diff
@@ -65,7 +65,8 @@ import scala.collection.immutable.Queue
 
 val renderer = Renderer(
   renderingOptions = RenderingOptions(density = 75),
-  directory = Paths.get("images", "usage")
+  directory = Paths.get("images", "usage"),
+  format = PNG  // or JPG
```

Comment:
1. This file is auto-generated. Could you update [the source](https://github.com/stanch/reftree/blob/master/demo/src/main/tut/README.md) and run `sbt tut` instead?
2. This change seems outdated, should be `format = "png"`.
3. I would prefer to show the default usage first, without specifying the format explicitly, and then add another example with the format specified. The explicit format can be something other than PNG, for example `"gif"` (please, not `"jpg"` :) ). Would be nice to add a comment saying that any format listed by `dot -T?` can be used.